### PR TITLE
Add new Java Math Methods entry: `.sqrt()` method

### PR DESCRIPTION
**Pull request by Yash-Var**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Sqrt'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.sqrt()`** method returns the ` Square root ` of a number.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:15]:
```suggestion
The **`Math.sqrt()`** method returns the square root of a double value passed to it as argument.
```

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+Title: '.sqrt()'
+Description: 'Returns the square root of a value of type double passed to it as argument.'
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:3]:
```suggestion
Description: 'Returns the square root of a double value passed to it as argument.'
```

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The **`Math.sqrt()`** method returns the ` Square root ` of a number.
+
+## Syntax
+
+```pseudo
+Math.sqrt(double num)
+```
+
+The `.sqrt()` method takes one parameters.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:23]:
```suggestion
The `.sqrt()` method takes one parameter.
```

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Example
+
+To show working of `Math.sqrt()` method.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:27]:
```suggestion
This example shows the working of the `Math.sqrt()` method.  
```

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  public static void main(String args[]) {
+       double x = 64.0;  
+       System.out.println(Math.sqrt(x));
+       double y = 49.0;
+       System.out.println(Math.sqrt(y));
+   }
+}
+```
+
+## Output:
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:40]:
```suggestion
This results in the following output:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var, Thanks for your contribution! I did the first review and left a few comments.

Please remember to sign the CLA.

**Comment by Yash-Var** [comment]
@SSwiniarski what i do next ......

**Comment by SSwiniarski** [comment]
@Yash-Var, first you should [sign the CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=936) which is a requirement for contributing. Then you should review my comments and either commit the suggestion or edit your entry directly in response to the editorial feedback. Tag me in a comment here when you're done editing your entry.

**Comment by Yash-Var** [comment]
@SSwiniarski completed pls review

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+Title: '.sqrt()'
+Description: 'Returns the square root of a double value passed to it as an argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Sqrt'
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:9]:
```suggestion
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
```

For the meta-`Tags`, let's use what's inside the [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. I suggest adding `'Functions'` for SEO-purposes, as well as the `'Numbers'` tag.

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.sqrt()`** method returns the square root of a double value passed to it as an argument.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:15]:
```suggestion
The **`Math.sqrt()`** method returns the positive, properly rounded square root of a double-type value.
```

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+## Syntax
+
+```pseudo
+Math.sqrt(double num)
+```
+
+The `.sqrt()` method takes one parameter.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:23]:
```suggestion
The return type of the `.sqrt()` method is a double. The data type of the `num` parameter is also a double. However, if a decimal is not provided, `num` will still be treated like a double-type.

Some edge cases for the `.sqrt()` method include the following:

 - If `num` is less than zero or it is not a numeric type (`NaN`), then `NaN` is returned.
 - If `num` is positive infinity (`Double.POSITIVE_INFINITY`), then `Infinity` is returned. 
```

The "Syntax" section is a good place to list out, if any, the edge cases of parameters used in a method/function.

File: content/java/concepts/math-methods/terms/sqrt/sqrt.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+## Example
+
+This example shows the working of the `Math.sqrt()` method.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sqrt/sqrt.md:27]:
```suggestion
The following example showcases the `.sqrt()` method being applied to two variables:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Yash-Var First of all, I want to thank you very much for the quick turnaround on this rewrite. This is much better! 😄 🚀

I've finished my review and left you some comments/suggestions to look over. Let us know if you have any questions and tag either myself or @SSwiniarski to let us know you've implemented the new changes.

**Review by SSwiniarski — APPROVED** [review]

**Review by Dusch4593 — APPROVED** [review]
@Yash-Var This entry is looking good to me! 😄 To get the GitHub checks to pass, I removed extra/trailing spaces in 57a715656f08e682fca63087463fd6bebc75baaa and 98ffe448f478156076320def42baa67d9533669a.

This entry looks ready to merge (🚀) but a few things for you to note for your next pull request:

- Try not create new folders/files in the `main` branch of your forked Docs repo. Instead, we recommend creating a new Git branch and creating/pushing commits from there. The name of the PR branch should reflect the work being done (in this case, I would have named this one `java-math-sqrt`). 
  - The way I do this is with `git checkout -b [name-of-new-branch]` where I create and check out to the new branch in one command.
  - Alternatively, you could run `git branch [name-of-new-branch]` followed by `git checkout [name-of-new-branch]`.
- When you create the new pull request here on GitHub, make sure to review and check the boxes (I went ahead and checked them since this PR is about to be merged):
Screen Shot 2022-09-03 at 10 30 33 AM

**Comment by Dusch4593** [comment]
On another note, congratulations @Yash-Var for your first contribution to Docs. The entry is now live here: https://www.codecademy.com/resources/docs/java/math-methods/sqrt

Something I noticed, though. Your profile isn't showing under the list of Contributors for this entry:

Screen Shot 2022-09-03 at 10 38 25 AM

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

After this, I recommend logging out of your Codecademy account and re-logging in through GitHub:

Screen Shot 2022-08-22 at 6 46 51 PM

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)

**Comment by Yash-Var** [comment]
@Dusch4593 @SSwiniarski thanks for guidance .....

**Comment by Dusch4593** [comment]
Screen Shot 2022-09-03 at 11 40 52 AM

There you are! 😄

**Pull request merged**